Description: Correct "normalize" program and package name.
 "normalize" Debian package was renamed to "normalize-audio".
Author: Maxime Chatelle <xakz@rxsoft.eu>
Bug-Debian: http://bugs.debian.org/589790
Origin: vendor
Forwarded: not-needed
Last-Update: 2018-06-16
--- a/pacpl.in
+++ b/pacpl.in
@@ -2572,8 +2572,8 @@
        }
        
        # normalize wav file before encoding to output format
-       perror("no_app","normalize") if $normalize and not `which normalize 2>/dev/null`;
-       system("normalize $nopts \"$outf.$$.wav\"") if $normalize;
+       perror("no_app","normalize") if $normalize and not `which normalize-audio 2>/dev/null`;
+       system("normalize-audio $nopts \"$outf.$$.wav\"") if $normalize;
 
        # copy meta-data from input file to %tag_name 
        read_tags("$inf.$infmt", "$codec") if $run{$codec}{TAGS}{READ} == 1 and not $dryrun;
